unit_tests/CodeBlockTests.d

```diff
@@ -0,0 +1,169 @@
+/**
+ * Copyright: Copyright (c) 2016 Wojciech Szęszoł. All rights reserved.
+ * Authors: Wojciech Szęszoł
+ * Version: Initial created: Feb 14, 2016
+ * License: $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost Software License 1.0)
+ */
+
+import std.stdio;
+import Common;
+import dstep.translator.CodeBlock;
+
+unittest
+{
+    assertEq("\n", compose(CodeBlock("", EndlHint.Group, [])));
+
+    auto declInt = CodeBlock("int a;", EndlHint.SingleLine, []);
+    auto declFloat = CodeBlock("float b;", EndlHint.SingleLine, []);
+    auto group1 = CodeBlock("", EndlHint.Group, [declInt, declFloat]);
+
+    assertEq("int a;\nfloat b;\n\n", compose(group1));
+
+    auto declString = CodeBlock("string s;", EndlHint.SingleLine, []);
+    auto declFuncDecl = CodeBlock("void func();", EndlHint.SingleLine, []);
+    auto group2 = CodeBlock("", EndlHint.Group, [declString, declFuncDecl]);
+    auto group3 = CodeBlock("", EndlHint.Group, [group1, group2]);
+    assertEq(q"EOS
```

Comment:
When using delimited strings I prefer that the identifier is matching the language in the code is in. Some editors might highlight that.